CONTRIBUTING.md

```diff
@@ -1,35 +1,111 @@
+
+[gfi]: https://github.com/facebook/react-native/labels/good%20first%20issue
+[docs-prs]: https://github.com/facebook/react-native-website/pulls
+[pr-no-test-plan]: https://github.com/facebook/react-native/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+-label%3A%22PR%3A+Includes+Test+Plan%22+
+<!-- END: Ways to Contribute -->
+
+
+### Our Development Process
+
+Most changes from engineers at Facebook will sync to [GitHub][facebook/react-native] through a bridge with Facebook's internal source control. Changes from the community are handled through GitHub pull requests. Once a change made on GitHub is approved, it will first be imported into Facebook's internal source control. The change will eventually sync back to GitHub as a single commit once it has passed Facebook's internal tests.
+
+[facebook/react-native]: https://github.com/facebook/react-native
+<!-- END: Development Process -->
+
+### Repositories
+The main repository, <https://github.com/facebook/react-native>, contains the core React Native framework, and it is here where we keep track of bug reports and manage pull requests.
+
+There's a few other repositories you might want to familiarize yourself with:
+
+- The source code for the **React Native website**, including the documentation, is located at <https://github.com/facebook/react-native-website>
+- **Releases** are coordinated through the <https://github.com/react-native-community/react-native-releases> repository. This includes important documents such as the Changelog.
+- **Discussions** about the future of React Native take place in the <https://github.com/react-native-community/discussions-and-proposals> repository.
+
+Browsing through these repositories should provide some insight into how the React Native open source project is managed.
+<!-- END: Repositories. -->
+
+<!-- START: GitHub Issues -->
+# GitHub Issues
+
+We use GitHub issues to track public bugs. Please ensure your description is clear and has sufficient instructions to be able to reproduce the issue.
+
+## Security Bugs
+
+Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe disclosure of security bugs. In those cases, please go through the process outlined on that page and do not file a public issue.
+<!-- END: GitHub Issues -->
+
+# Helping with Documentation
+
+The React Native documentation is hosted as part of the React Native website repository at https://github.com/facebook/react-native-website. The website itself is located at https://facebook.github.io/react-native and it is built using [Docusaurus](https://docusaurus.io/). If there's anything you'd like to change in the docs, you can get started by clicking on the "Edit" button located on the upper right of most pages in the website.
+
+If you are adding new functionality or introducing a change in behavior, we will ask you to update the documentation to reflect your changes.
+
+## Contributing to the Blog
```

Comment:
Are we looking to actively include external blog posts on the website?

Reply:
@cpojer might know